### ISSUE TITLE
Overhaul Saar.

### ISSUE THREAD

**Issue by DaloLorn** [Alarcarr/Rising-Star#32]
From the Discord chat:

- [ ] Change homeworld:
  - [ ] Has some kind of 'holy' planetary engine, whatever that means, and vast plains
  - [ ] Levels up based on amount of evangelized planets
  - [ ] Receives various boosts based on amount of evangelized planets
  - [ ] Is immune to annex votes. (But not to diplomatic victory, I hope? Maybe instead of this they should be immune to defeat for as long as they own a Star Temple or evangelized planet...)
- [ ] Broader gameplay:
  - [ ] Does not colonize planets. (How the hell are we going to make them win wars, though... Ringworlder-style depopulation does not seem applicable here.)
  - [ ] Evangelizes other empires' planets to gain resources, preferably dependent on the resources being produced by that planet
  - [ ] Declaring war will cause all evangelized planets owned by the attacker to be debuffed somehow (inverted Happy was Darloth's first idea)
  - [ ] Missionaries extend evangelization duration if evangelized by their owner, or overwrite evangelization otherwise (I was more in favor of using some kind of 'Zeal' meter, forcing people to make Star Temples to increase their Zeal, losing Zeal as they evangelized more planets or lost Star Temples, and then comparing empires' Zeal meters to determine the victor in situations like these)
  - [ ] Gives boosts to evangelized allies? Happy-like status or something?

The general idea is that they're more spiritual than materialistic in their efforts, a race that tries to manipulate people in a very [Prior-like](http://stargate.wikia.com/wiki/Prior) way in preparation for the Overlord invasion. A sharp contrast to the Feyh and their 'believe or die!' doctrine, at least for now - that one is intended to weaken the enemies instead of creating allies.

**Comment by DaloLorn** [comment]
Optional, possibly in need of balance discussion but it seems to fit with the lore:

- [ ] Centralized races like Oko and Hoonan benefit less from evangelization due to the way they function, but also get less of a debuff for the same reason. (Naturally, the Saar also get less of a buff from them.) Not sure if Oko should be affected more or less severely than the Hoonan.
- [ ] Mono can't be evangelized? Not sure about this - on one hand, their programming could conceivably prevent such things, but depending on how much free will they have... it could be just as easy as with a regular flesh-and-blood sentient.

Also don't know how to treat Atroans.

**Comment by darloth** [comment]
Regarding other species and their vulnerability or not to evangelism - It's probably best to avoid messing with a core mechanic of a race if we can avoid it.  Even if lore-wise, it makes sense for certain races to be harder or easier to influence, that turns race selection into a counter-pick scenario, which is usually a bad idea for 4x or RTS games.  For certain, balancing that would be very difficult and you could easily end up with scenarios where a Saar player is stuck between two others who they can do almost nothing about.

**Comment by Alarcarr** [comment]
What about rhata ....

**Comment by DaloLorn** [comment]
Darloth: I was thinking no more than a 50% debuff, tops - probably less. In any case, I've been thinking of making some kind of 'lore mode' or something anyway by the time the Overlords come out. Also a nice bunch of campaigns to go along with it, if I can figure out how to generate a good map.

Alar: Hmm?

**Comment by Alarcarr** [comment]
Before we open new horizons i think we need to fix what we already have to bea ablue to push to steam a version.And rhata is still half-done.

**Comment by DaloLorn** [comment]
Yeah, sure. If you'd read the chat you'd know Darloth and I already concluded that this isn't for the next version or two at the very least. :P
